examples/run-blockchain/index.ts

```diff
@@ -39,10 +48,12 @@ async function main() {
 
   await vm.runBlockchain(blockchain)
 
-  const blockchainHead = await promisify(vm.blockchain.getHead.bind(vm.blockchain))()
+  const blockchainHead: IBlockchainHead = await promisify(
+    vm.blockchain.getHead.bind(vm.blockchain),
+  )()
 
   console.log('--- Finished processing the BlockChain ---')
-  console.log('New head:', '0x' + blockchainHead.hash().toString('hex'))
+  console.log('New head:', '0x' + blockchainHead.header.hash().toString('hex'))
```

Comment:
Is this for clarification purposes? `blockchainHead.hash()` should also work right?

Reply:
Yes it works if `hash` is added to the interface: 

```
interface IBlockchainHead {
  _common?: any
  transactions?: any
  uncleHeaders?: any
  _inBlockchain?: any
  txTrie?: any
  header?: any
  hash?: any
}
```

Let me know if it should be added, or if there is a better way than using the interface to fix this issue.